### DISCUSSION TITLE
Crowdin

### DISCUSSION THREAD

**Discussion by seerge** [seerge/g-helper#759] [Announcements]
Hello dear translators,

@marcelomijas @IceStormNG @hkaancaliskan @Constrat  @weslleykawan @hellotale @lswlc33 @Hoangdus @MDowski @etylix @TomBonnot @sklynic @martin6982 @utchka 

First of all, thanks for your help and effort with making app available in more languages ;)

Since it becomes more difficult to manage translations manually (and keep track of all newly added strings), I have moved them to a system called Crowdin, that should make whole process a bit easier.

You can join it by following link and then apply for your language, I will approve you: 
https://crowdin.com/project/g-helper

Thanks in advance. Serge.

**Comment by martin6982** [comment]
I just tried it this morning, it's much more convinent than before, I don't have to compare the difference between two strings files line by line now, it's really great. thank you so much!

**Comment by Czompi** [comment]
Hello @seerge,

Could you please add Hungarian language to Crowdin?

Thanks in advance,
Czompi

**Reply by seerge** [reply]
Added

**Reply by seerge** [reply]
@Czompi 

Thanks for the input, I would try to make all labels as short as possible (as they sometimes don't fit into layout / cut into 2 lines / etc) :)
Btw, translation doesn't have to be literal, as far as it maintains main sense / message :) 

![Screenshot 2023-07-13 110732](https://github.com/seerge/g-helper/assets/5920850/6c5b087f-f2f6-4968-b4a8-75ab80c8f0fe)

Btw, this applies to other languages as well (especially to other fresh translations , like @yungsinatra0 ;)

**Reply by yungsinatra0** [reply]
@seerge is there any way for us to see how the translated text looks?

**Reply by seerge** [reply]
Only by compiling app unfortunately :) i add / merge all changes from crowdin like once a day or so to github
[GHelper.zip](https://github.com/seerge/g-helper/files/12037637/GHelper.zip)

**Reply by Czompi** [reply]
I started tweaking it, but hadn't had a time to test it on my laptop. We call car's "turbo" as "turbó" in Hungarian and ASUS' local site also calls it "turbó". Also I'm working on aligning my translations with Armoury Crate's.

**Comment by MDowski** [comment]
Amazing! It is indeed much easier to work on this now. Thank you for this. ❤️

Could you please clarify, if we have some thoughts and doubts, shall we call you with @seerge in every comment, or only in the most important ones?

**Reply by seerge** [reply]
Great, I just didn't get how so many people managed to join without my approval so far, haha (did you use a link from email? or just joined from public project page) ?

You can mention me every time you have questions ofc.

**Reply by Constrat** [reply]
I think it's possible that by going to crowdin with a GitHub account it automatically approves?

**Reply by MDowski** [reply]
I've logged in with my GitHub account and I was indeed approved manually.

**Reply by martin6982** [reply]
I did used the link in the email.

**Reply by martin6982** [reply]
@seerge I check the activity history in Crowdin, maybe it's because you changed the project's the visibility level to "public" ?

**Reply by seerge** [reply]
@martin6982 it has to be public in order to add translators (private projects need to have paid subscription to do that)

**Comment by martin6982** [comment]
> @martin6982 it has to be public in order to add translators (private projects need to have paid subscription to do that)

oh I see

**Comment by TomBonnot** [comment]
Great idea @seerge I ddin't have time to make a new fork haha 
I did join as RoutardTL

**Comment by yungsinatra0** [comment]
Do we have to be certified translators to contribute? I'd love to contribute to the Romanian language translation if possible!

**Reply by seerge** [reply]
@yungsinatra0 common sense is enough :) i have added Romanian to the list

**Comment by mapm1998** [comment]
Hello @seerge. I've noticed that the portuguese translation on G-Helper is for the brazilian portuguese, and not the european one. First of all, how does this work, the contribution to the translation, do we need to ask for any kind of permition?
Also, with the portuguese translation being the brazilian one, does it justify the second translation to the european one? If that's the case, I'm more than avaliable to to the translation.

**Reply by TomBonnot** [reply]
https://crowdin.com/project/g-helper Go to this page, sign up and start modifying a language (in your case PT-pt) 
Tho I would be very interested in which way you could make it PT-pt and not PT-br (since I am learning it and I have worked on the translation with a Brazilian friend of mine) ! 

I do think it could be an option. I mean, it is one file, it won’t hurt the stability of the project or anything, nor make it heavier so. Go on ! I will (I hope I can) Modify the PT-pt into PT-br and let you create another. Or maybe we will need the help of Seerge

**Reply by seerge** [reply]
@mapm1998 before it was only 1 translation, that was called PT-pt and it was used for all versions of Portuguese :) And @TomBonnot helped initially with that

I have just created a second locale in https://crowdin.com/project/g-helper for PT-br (which is empty) , and PT-pt is apparently half-filled with Brazilian version :) 

You can sign up on crowdin website and just join the project and translate it there. Except, I don't know how to move existing PT translation to BR branch on crowding.

Given that existing translation covers only part - can you guys just check both versions and fill everything? (if you want of course) :)

**Reply by TomBonnot** [reply]
@seerge can’t you rename the first one into PT-br and then re create a PT-pt ? 

Otherwise I will copy paste everything to let you start translations @mapm1998

**Reply by seerge** [reply]
I have managed to copy pt-PT to pt-BR

So they are same now

@mapm1998 can translate (by fixing wrong lines) in pt-PT

and @TomBonnot can finish pt-BR :)

![Screenshot 2023-08-20 172653](https://github.com/seerge/g-helper/assets/5920850/34ec7191-0299-4052-8b40-7bafc36bd42a)

**Reply by TomBonnot** [reply]
Cool Thank you @seerge !

**Reply by mapm1998** [reply]
Of course I can! Thank you @seerge !

**Comment by Kikimahe** [comment]
Hello Seerge,

I wanted to translate these words, but it can't be found on Crowdin. :)

![Capture d'écran 2024-01-31 203441](https://github.com/seerge/g-helper/assets/142881210/8e715df9-1b69-421b-bc29-b5dadbf89173)

**Reply by seerge** [reply]
@Kikimahe it's not there, same as some other newly added things. I will add them to translations somewhere soon.

**Reply by Kikimahe** [reply]
Ah ok. 👍 

Thanks.
Good evening.

**Comment by alextrik67** [comment]
Hi, could you add Russian language to Crowdin?
